bitnami/spring-cloud-dataflow/values.yaml

```
@@ -134,6 +134,10 @@ server:
   ##
   podAntiAffinityPreset: soft
 
+  ## Dataflow Server port
+  ##
+  port: 8080
```

Use `containerPort` instead:
```suggestion
  ## Dataflow Server port
  ##
  Use `containerPort: 8080
```